demos/smart_classroom_demo_gapi/create_list.py

```diff
@@ -41,4 +41,4 @@
         objects[label] = [path]
 
 with open('faces_gallery.json', 'w') as outfile:
-    json.dump(objects, outfile, indent=4)
+    json.dump(objects, outfile, indent=4)
```

Comment:
Please revert

Reply:
Fixed. File is identical to the current version in OMZ now.